src/NuGet.Clients/NuGet.PackageManagement.UI/Xamls/PackageItemControl.xaml

```diff
@@ -190,22 +190,29 @@
                       Grid.Row="2">
           <Grid.RowDefinitions>
             <RowDefinition Height="auto" />
-            <RowDefinition Height="4" />
             <RowDefinition Height="*" />
+            <RowDefinition Height="auto" />
           </Grid.RowDefinitions>
           <!-- providers -->
           <nuget:PackageManagerProvidersLabel
                         Grid.Row="0"
                         FormatString="{x:Static nuget:Resources.Label_UseInstead}"
                         DataContext="{Binding Providers}" />
-
           <!-- Summary -->
           <TextBlock
-                        Grid.Row="2"
+                        Grid.Row="1"
                         Text="{Binding Summary, Converter={StaticResource SummaryToStringConverter}}"
                         FontStyle="{Binding Summary, Converter={StaticResource SummaryToFontStyleConverter}}"
                         TextTrimming="CharacterEllipsis"
                         TextWrapping="Wrap" />
+          <!-- Deprecation message -->
+          <nuget:PackageItemDeprecationLabel
+                        Grid.Row="2"
+                        Visibility="{Binding IsPackageDeprecated, Converter={StaticResource BooleanToVisibilityConverter}}"
+                        FormatStringSingle="{x:Static nuget:Resources.Deprecation_PackageItemMessage}"
+                        FormatStringAlternative="{x:Static nuget:Resources.Deprecation_PackageItemMessageAlternative}"
+                        DataContext="{Binding DeprecationMetadata}" />
```

Comment:
nit: indentation is quite far in. I noticed this file has an inconsistent mix of proper indentation and really far indentation.

Reply:
I tried to keep indentation style from previous lines. Let me know if this need change.

Reply:
I will the current indentation style. Hope to unify identation style across all XAML/XML files in NuGet in another PR.